CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 1.14.0 01-03-2018
+
+- add --worker-processes argument
+- Fix a bug in nginx-auto.conf.template
+- scale nginx workers with number of cores
```

Comment:
please combine 1 and 3, they are the same change. 
remove 2

Reply:
done